packages/blade/src/components/Input/DropdownInputTriggers/__tests__/AutoComplete.web.test.tsx

```diff
@@ -387,27 +399,28 @@ describe('<BottomSheet /> & <Dropdown /> with <AutoComplete />', () => {
       </Dropdown>,
     );
 
-    const [selectInput, autocomplete] = getAllByLabelText('Fruits');
+    const selectInput = getByLabelText('Fruits') as HTMLInputElement;
+    const autoComplete = queryByTestId('fruits-autocomplete') as HTMLInputElement;
     expect(selectInput).toBeVisible();
 
     // testing library ignores the nodes because they are set to display none so using querySelector to select from dom instead.
     // the node becomes accessible after click on selectInput
     expect(queryByTestId('bottomsheet-body')).not.toBeVisible();
     await user.click(selectInput);
     await waitFor(() => expect(queryByTestId('bottomsheet-body')).toBeVisible());
-    expect(autocomplete).toBeVisible();
+    expect(autoComplete).toBeVisible();
     expect(container).toMatchSnapshot();
   });
 
   it('should handle AutoComplete behaviour in single select', async () => {
     const user = userEvent.setup();
 
-    const { getByRole, queryByRole, getAllByLabelText, queryByTestId } = renderWithTheme(
+    const { getByRole, queryByRole, getByLabelText, queryByTestId } = renderWithTheme(
       <Dropdown>
-        <AutoComplete label="Cities" placeholder="Select Cities" />
+        <SelectInput testID="select" label="Cities" />
```

Comment:
We changed this from AutoComplete to SelectInput???

Reply:
@anuraghazra This should have been SelectInput, Autocomplete is used below.

Reply:
@saurabhdaware Can you once confirm if 2 Autocomplete here are intended?

Reply:
Oh weird, @saurabhdaware can you confirm if this was a bug?

Reply:
Oh yes right it should be SelectInput since we are using AutoComplete inside BottomSheet below.